docs/static/swagger.yaml

```diff
@@ -18774,6 +18795,19 @@ definitions:
 
               are sent to
             type: string
+          whitelisted_hooks:
+            items:
+              properties:
+                code_id:
+                  format: uint64
+                  type: string
+                denom_creator:
+                  type: string
+              type: object
+            title: >-
+              HookWhitelist is the list of hooks which are allowed to be added
```

Comment:
might be missing something, but why do we have `whitelisted_hooks` as the key here and `HookWhitelist` as the value?

Reply:
ah need to regen swagger